pkg/operator/ceph/cluster/cephstatus_test.go

```diff
@@ -197,3 +203,104 @@ func Test_cephStatusChecker_conditionMessageReason(t *testing.T) {
 		})
 	}
 }
+
+func TestForceDeleteStuckRookPodsOnNotReadyNodes(t *testing.T) {
```

Comment:
This test should also verify that non-matching pods are not deleted.

Reply:
updated